packages/css/src/plugins/atomicify.tsx

```diff
@@ -28,8 +28,8 @@ const normalizeSelector = (str?: string) => {
 
 const atomicifyDecl = (node: Declaration, opts: Opts = {}) => {
   const normalizedSelector = normalizeSelector(opts.parentSelector);
-  const group = hash(`${join(opts.parentAtRule, opts.parentSelector)}-${node.prop}`);
-  const className = `cc-${group}-${hash(node.value)}`;
+  const group = hash(`${join(opts.parentAtRule, opts.parentSelector)}${node.prop}`).slice(0, 4);
+  const className = `${group}${hash(node.value).slice(0, 4)}`;
```

Comment:
It looks like this change would  break the ax runtime - which splits the class name on the `-` in order to identify the group (for dedupe/handling overridden styles).

Reply:
good catch! yeah you're right - we'd be changing `ax` to get the first 4 characters instead